src/s2e/Plugins/Searchers/SeedSearcher.cpp

```diff
@@ -469,7 +471,11 @@ void SeedSearcher::handleGetSeedFile(S2EExecutionState *state, S2E_SEEDSEARCHER_
 
     m_selectSeedState = false;
 
+#if defined(CONFIG_SYMBEX) && defined(CONFIG_SYMBEX_MP)
     cmd.GetFile.Result = 2;
+#else
+    cmd.GetFile.Result = 3;
```

Comment:
Could you use a constant instead of a magic number? I know there was one before, but looks like a good time to refactor.